engineering-progression-framework.md

```diff
@@ -0,0 +1,143 @@
+# Engineering Progression Framework
+
+This framework outlines what we expect from programmers in terms of technical skill, level of impact, and the manner in which you conduct your work. It is not a checklist! It's a guide to facilitate ongoing conversations around performance and progression between programmer and manager.
+
+We use the framework to guide personal development, honing in on focus areas to accelerate progression in your 37signals career. These personal development conversations happen between you and your manager, throughout the year, alongside regular and specific feedback. The framework also acts as a foundation for our annual performance reviews which are aligned with your work anniversary.
+
+37signals level pay with the top 10% of the industry (based on San Francisco rates), so we expect the quality of the work itself to be commensurate with that target.
+
+
+## Junior Programmer
```

Comment:
@alarowe FYI I made a tabular version of this but it looked terrible on GitHub because they have a fixed-width layout